xlators/cluster/afr/src/afr-open.c

```diff
@@ -295,7 +510,7 @@ afr_fix_open(fd_t *fd, xlator_t *this)
 
     need_open = alloca0(priv->child_count);
 
-    call_count = afr_fd_ctx_need_open(fd, this, need_open);
+    call_count = afr_fd_ctx_get_need_open(fd, this, need_open);
```

Comment:
When we execute a command like
`dd if=/dev/zero of=1 bs=1M count=100` etc 8 parallel writes would be executed on the fd. This can lead to 8 afr_fix_open() calls. Which will lead to 8 getlks here. Is that necessary?

Reply:
In a case when there is lock held & one brick needs re-open and these 8 parallel write comes, during the first 2 writes it will not re-open due to both strict-locks setting and locks-held are true. Now the client releases the lock, at that time the 3rd write can continue with re-open and the write could succeed if we do getlk for all the writes. Otherwise it will still fail.